app/generators/cinema_halls/generate_seats.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module CinemaHall
+  module Generators
```

Comment:
You created here hierarchy:
1. CinemaHall
2. Generators
3. GenerateSeats

Why does your directory tree look like this?
1. app
2. generators
3. cinema_halls
4. generate_seats

How it's connected? Or maybe it's completely not connected?

Reply:
Generators folder will have folders with models (later: movies, tickets, reservations...) In each of them I want to have files having methods like create. Generate_seats should belongs_to cinema_hall, because it generates seats for this model.